### ISSUE TITLE
Devices mid_d0001/elt/master failing

### ISSUE THREAD

**Issue by HFRibeiro** [ska-telescope/tmc-prototype#25]
The mid_d0001/elt/master device if failing after some time of use, it keeps down until we manually restart the devices:
Example Log:
```
Device:  mid_d0001/elt/master  Attribute:  achievedPointing
Polling Period:  1000
Traceback (most recent call last):
 File "/app/tmcprototype/configurePollingAndEvents.py", line 25, in <module>
   attributeProxy.poll(attributeProperty["pollingPeriod"])
 File "/venv/lib/python3.5/site-packages/tango/attribute_proxy.py", line 333, in __new_fn
   return getattr(self._AttributeProxy__dev_proxy, dp_fn_name)(self.name(), *args, **kwds)
PyTango.DevFailed: DevFailed[
DevError[
   desc = Failed to connect to device mid_d0001/elt/master
          The connection request was delayed.
          The last connection request was done less than 1000 ms ago
 origin = Connection::reconnect
 reason = API_CantConnectToDevice
severity = ERR]
]
```
This also makes all the other devices fail

**Comment by jyotinpo** [comment]
Thanks for reporting the issue.
We have observed this issue when there is some misconfiguration with the hostname/network. 
Can you share the steps in order to reproduce this issue?
The other devices will fail as we have added the dependency on DishMaster.
I will try running the devices for a couple of days to see if I can reproduce the issue.

**Comment by HFRibeiro** [comment]
Thanks for the quick response,
To reproduce the issue, you can just from a clean linux machine do the following:

- Install docker and docker compose
- Install `make` 
- Clone https://github.com/ska-telescope/ska-engineering-ui-compose-utils
- Inside the repo, run `make up` and them `make tmc`

This will create an env, based on the current docker images on the nexus repo, all the TMC prototype ones, and webjive.

Then after everything is finished you can access [http://localhost:22484/testdb](http://localhost:22484/testdb) and you should have webjive running with TMC

As this is not an action/reaction type of error, I can't really tell how to trigger this. For now, it happened randomly with diferents amounts of time and giving the log reported above.

**Comment by jyotinpo** [comment]
Hi @HFRibeiro,
Are you facing this issue now? I assume that was some misconfiguration of the network in the environment. We recently followed your steps as a part of System Demo-2 (PI3) in one of the local machine. Everything seems to be running fine. I have also tried running the updated `tmc-prototype` in the K8s integration environment and everything seems to be up and running. Are we good to close this issue?
